node/src/actors/chain_manager/mod.rs

```diff
@@ -1460,7 +1459,6 @@ impl ChainManager {
             ),
         }
 
-        self.notify_node_status(next_state);
```

Comment:
Breaking change, the node should still support "status" subscriptions even if the wallet is not using it.